.github/workflows/deploy-frontend-nuxthub.yml

```
@@ -0,0 +1,228 @@
+    runs-on: ubuntu-latest
+    needs: [deploy-production]
+    if: always()
+    steps:
+      - name: Send Slack notification
+        uses: 8398a7/action-slack@v3
+        with:
+          status: ${{ job.status }}
+          text: |
+            Frontend Deployment ${{ job.status }}
+            Workflow: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
+          webhook_url: ${{ secrets.SLACK_WEBHOOK }}
```

actionlint (1.7.8)

input "webhook_url" is not defined in action "8398a7/action-slack@v3". available inputs are "author_name", "cancelled_message", "channel", "custom_payload", "failure_message", "fields", "github_base_url", "github_token", "icon_emoji", "icon_url", "if_mention", "job_name", "mention", "status", "success_message", "text", "username"

(action)